server/src/main/java/org/opensearch/action/support/clustermanager/ClusterManagerNodeOperationRequestBuilder.java

```diff
@@ -59,7 +59,7 @@ protected ClusterManagerNodeOperationRequestBuilder(OpenSearchClient client, Act
      */
     @SuppressWarnings("unchecked")
     public final RequestBuilder setMasterNodeTimeout(TimeValue timeout) {
```

Comment:
Should the `setMasterNodeTimeout` method names in this class be changed?

Reply:
Thanks! You are so careful, I can't believe this is missed. I will add this method name to the PR description and rename it.

Reply:
Changed in the new commit https://github.com/opensearch-project/OpenSearch/pull/3617/commits/a77d350f727579f6f7a9acd5bcd3e474d8cbc615